porting/debug-build/wifi.rst

```diff
@@ -11,18 +11,24 @@ Testing wifi is done using NetworkManager
 
    nmcli d
 
-Qcom devices
-------------
+Qcom devices and CAF devices
+------------------------------------
 
-Wifi is fairly easy to get going on most Qcom based devices
+Wifi is fairly easy to get going on most Qcom and CAF based devices
 
 .. code-block:: guess
 
-   echo 1 > /dev/wcnss_wlan
+   echo 1 > /dev/wcnss_wlan #if not up by default
```

Comment:
I would prefer if the note was written above the section (not via an in-line comment). Also, it's not clear what the note means to newcomers